roles/openshift_docker/tasks/main.yml

```diff
@@ -24,5 +27,5 @@
   - role: docker
     local_facts:
       openshift_image_tag: "{{ l_image_tag }}"
-      openshift_version: "{{ l_image_tag if l_image_tag is defined else '' | oo_image_tag_to_rpm_version }}"
```

Comment:
l_image_tag is only defined when openshift.common.is_containerized is true you probably want:

```
openshift_image_tag: "{{ l_image_tag | default(None) }}"
```